python/ray/tests/test_basic_2.py

```diff
@@ -720,6 +720,46 @@ def get(self):
         assert b"OK" in output
 
 
+@pytest.mark.skipif(client_test_enabled(), reason="init issue")
+def test_use_dynamic_function_and_class():
+    # Test use dynamically defined functions
+    # and classes for remote tasks and actors.
+    # See https://github.com/ray-project/ray/issues/12834.
+
+    current_path = os.path.dirname(__file__)
+    job_config = ray.job_config.JobConfig(code_search_path=[current_path])
+    ray.init(job_config=job_config)
+
+    def foo1():
+        @ray.remote
+        def foo2():
+            return "OK"
+
+        return foo2
+
+    @ray.remote
+    class Foo:
+        @ray.method(num_returns=1)
```

Comment:
Out of curiosity, what is this line for?  Does it work without this line?

Reply:
Yes, it's ok to drop @ray.method(num_returns=1).